test-data/unit/check-overloading.test

```diff
@@ -1412,15 +1412,19 @@ reveal_type(f(d))  # E: Revealed type is 'builtins.list[builtins.int]'
 from typing import overload, Any
 
 @overload
-def f(*, x: int = 3, y: int = 3) -> int: ...
+def f(*, x: int = 3, y: int = 3) -> int: ...  # E: Overloaded function signatures 1 and 2 overlap with incompatible return types
```

Comment:
Hm, I don't think this this should be an error. The only call that is allowed for both is `f()`.